src/components/loginForm/loginForm.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <form action="" class="login-form">
+    <div class="input-text-row">
+      <input id="username"
+        name="username"
+        type="text"
+        class="input-text login-form__input">
+      <label for="username"
+        class="input-text-label">Username</label>
+    </div>
+    <div class="input-text-row">
+      <input id="password"
+        name="password"
+        type="text"
+        class="input-text login-form__input">
+      <label for="password"
+        class="input-text-label">Password</label>
+    </div>
+    <div class="login-form__footer">
+      <button class="btn btn--primary login-form__btn"
+        type="submit"
+        name="button">Login</button>
+      <a href=""
+        class="btn btn--link login-form__btn login-form__btn--password">Forgot your password ?</a>
```

Comment:
seems a lot of classes for only a link ...

Reply:
It's call BEM

Reply:
It's because I use both global classes for styles and local classes for spacings... I haven't found a more graceful way to do it yet.